cypress/e2e/Bugfix/Issue392.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Fix issue #392: Delete diagram should only remove parsable files of diagrams
+
+  Scenario: Delete diagram should remove it from database
```

Comment:
Its not really  "from database" but "from storage", no ?

Reply:
done